source/frontend/src/components/maps/leaflet/Control/LayersControl/LayersMenu.test.tsx

```diff
@@ -0,0 +1,44 @@
+import { createMemoryHistory } from 'history';
+import { noop } from 'lodash';
+import React from 'react';
+import { MapContainer } from 'react-leaflet';
```

Comment:
@buddy326 you have got some linter warnings on this file, please clean those up.